File: PiFrontEnd/src/app/model/RendezVous.ts
import {user} from '../model/User';
import {AnnonceCollocation} from '../model/AnnonceCollocation';
export class RendezVous{
    id!:number;
    date!: Date;
    status!: boolean;
    description!: string;
    owner!: user;
    client!: user;
    annonceCollocation!:AnnonceCollocation;
   IsitPassed!:boolean;
}

File: PiBackend/src/main/java/com/coco/pibackend/Entity/RendezVous.java
package com.coco.pibackend.Entity;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class RendezVous {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private Date date;
    private boolean status;
    @ManyToOne
    private User owner;
    @ManyToOne
    private User client;
    @ManyToOne

    private AnnonceCollocation annonceCollocation;

}
